addons/udes_stock/models/stock_picking.py

```diff
@@ -122,8 +126,9 @@ def _compute_picking_quantities(self):
                 if not has_discrepancies and qty_done != qty_todo:
                     has_discrepancies = True
 
-                total_qty_done += qty_done
-                total_qty_todo += qty_todo
+                factor = move.product_id.uom_id.factor
```

Comment:
If a package has 6 products, is the factor 1/6? Or 6? I'd imagine it has to be 1/6 if the u_total_quantity wants to be not in UOM, right?

Reply:
good spot